FileSpecs/Events.md

```diff
@@ -0,0 +1,265 @@
+|               |       | `06` | U06.1 | U06.2 | U06.3 | U06.4 | U06.5  | `00`  | U06.7   | `00`  | `00`  |
+|               |       | `07` | U07.1 | U07.2 | U07.3 | U07.4 | `00`   | `00`  | U07.7   | `00`  | `00`  |
+|               | 3D    | `08` | U08.1 | `00`  | `00`  | U08.4 | U08.5  | `00`  | `00`    | `00`  | `00`  |
+|               |       | `09` | U09.1 | U09.2 | U09.3 | U09.4 | U09.5  | U09.6 | U09.7   | U09.8 | U09.9 |
+|               |       | `0a` | U0a.1 | U0a.2 | `00`  | U0a.4 | `00`   | `00`  | `00`    | `00`  | `00`  |
+| **RestoreLP** | 2D    | `0b` | `00`  | `00`  | `00`  | *msg* | `00`   | `00`  | `00`    | `00`  | `00`  |
+| **RestoreSP** | 2D    | `0c` | `00`  | `00`  | `00`  | *msg* | `00`   | `00`  | `00`    | `00`  | `00`  |
+|               |       | `0d` | U0d.1 | `00`  | U0d.3 | `1a`  | U0d.5  | `00`  | `00`    | `00`  | `00`  |
+|               |       | `0e` | U0e.1 | U0e.2 | U0e.3 | U0e.4 | U0e.5  | U0e.6 | U0e.7   | U0e.8 | `00`  |
+|               |       | `0f` | U0f.1 | `01`  | U0f.3 | U0f.4 | U0f.5  | `00`  | `00`    | `00`  | U0f.9 |
+|               |       | `10` | U10.1 | U10.2 | `00`  | U10.4 | U10.5  | `00`  | U10.7   | `00`  | `00`  |
+|               |       | `11` | U11.1 | `00`  | U11.3 | U11.4 | U11.5  | `0`   | U11.7   | `00`  | `00`  |
+|               |       | `12` | U12.1 | U12.2 | U12.3 | U12.4 | `00`   | `00`  | U12.7   | `00`  | U12.9 |
+|               |       | `13` | U13.1 | U13.2 | `00`  | U13.4 | U13.5  | U13.6 | U13.7   | `00`  | U13.9 |
+| **Door3D**    | 3D    | `14` | `01`  | `00`  | `00`  | U14.4 | U14.5  | `00`  | U14.7   | `00`  | `00`  |
+|               | 2D    | `15` | U15.1 | U15.2 | U15.3 | `00`  | `00`   | `00`  | U15.7   | `00`  | `00`  |
+|               | 2D    | `16` | `00`  | `00`  | `00`  | `00`  | `00`   | `00`  | `00`    | `00`  | `00`  |
+| **Win**       | 2D    | `17` | `00`  | `00`  | `00`  | `00`  | `00`   | `00`  | `00`    | `00`  | `00`  |
+
+
+* Column `Where` indicates whether the event only appears in 2D or 3D maps.
+* Parameters of the form "U00.0" indicate an unknown parameter.
+* `00` indicates that this paramter always appears as `00`.
+
+This information is observational: the Amberstar *engine* might support more options values, but the *game* does not seeem to use them (unless events can be triggered in other ways than observed so far).
+
+<!-- ---------------------------------------- -->
+## Event 01: Teleport
+
+Teleports the party to the specified *map* - 1, coordinates (*x*, *y*).
+
+Unknown: *map*=0 might teleport the player to a different location in the same map (?)
+
+### Unknowns
+* U01.3: [`00-03`]
+
+<!-- ---------------------------------------- -->
+## Event 02: Door
+
+Party encounters a locked door.  The lock-picking difficulty seems to be indicated by *cr*, which is 1, 10, or 100.
+
+### Unknowns
+* Exact meaning of the challenge rating *cr*
+* U02.2: [`00`, `01`, `03` (only 3D maps)]
```

Comment:
In Ambermoon it is a percentage reduction of your lockpicking ability. 50 would reduce the success chance by 50. If you have 75 in lockpicking it would only be 25 for this lock.

The value 0 means (the door is open). But this is only used for chests I guess.

The value 100 means that it can't be lockpicked. This is used for doors and chests which require a key.

Reply:
Chests which should be lockpicked based on your skill have a value of 1 in Ambermoon. Most early chests have this value.

Reply:
That is consistent with the doors that I looked at, which is why I labelled it *cr*, though I am a bit confused that it only uses the values 1, 10, and 100...

Reply:
0 won't make sense for doors, 1 is the default for doors you can lockpick, 100 is for doors with keys and 10 is just a bit harder to lockpick.